storages/python/python-tests/bson_test.py

```diff
@@ -0,0 +1,73 @@
+"""Script to test the DLite plugin 'bson.py' in Python."""
+import os
+import shutil
+from pathlib import Path
+
+from run_python_tests import print_test_exception
+
+
+print('Running Python test <bson_test>...')
+thisdir = Path(__file__).absolute().parent
+input_path = thisdir / 'input'
+plugin = thisdir.parent / 'python-storage-plugins/bson.py'
+
+try:
+    with open(plugin, 'r') as orig:
```

Comment:
Instead of opening and changing the plugins, it is much easier to simply import them:

```python
plugins_dir = thisdir.parent / 'python-storage-plugins'
sys.path.insert(plugins_dir)

class DLiteStorageBase:
    """Dummy replacement for the built-in class in the Python plugins."""
    
import bson
```